stencil.lock

```diff
@@ -2,7 +2,7 @@ version: v1.43.0
 modules:
     - name: github.com/getoutreach/devbase
       url: https://github.com/getoutreach/devbase
-      version: v2.35.0-rc.1
+      version: malept/dt-5033/fix/mise-lintroller
```

Comment:
This will be replaced with a proper version once a stencil module upgrade happens.